### ISSUE TITLE
Precedence of the modulus operator

### ISSUE THREAD

**Issue by zyw727725828** [josdejong/mathjs#3189]
Why does 250/10%10 return the result Infinity? Why calculate % first and then /

**Comment by josdejong** [comment]
You can find the operator precedence here: https://mathjs.org/docs/expressions/syntax.html#precedence

You can check how an expression is evaluated via:

```js
math.parse('250/10%10').toString({ parenthesis: 'all'})
// "250 / (10 % 10)"
```

If you need an alterative order of exection, you can use parenthesis:

```js
math.evaluate('(250 / 10) % 10') 
// 5
```

**Comment by cirossmonteiro** [comment]
@josdejong I'm sorry for reviving this discussion, but according to the link provided before, division and modulus have the same precedence, which means the evaluation should be done from the left to the right.
Here's a screenshot from Javascript computation in Chrome's DevTools.
![image](https://github.com/josdejong/mathjs/assets/8466815/f7f25ce8-f5fd-4090-9f81-7c6b284c0f69)

**Comment by josdejong** [comment]
Thanks, you're right. I now see that the documentation is not up to date. The order of predencence of the modules operator was changed in #2304 I think, or maybe in consecutive changes around implementing special rules for implicit multiplication.

I've updated the docs to be consistent with the actual behavior: 599f4ee80482692c59a3e513472aaef77da075f8

Separately, we can discuss if we want to make changes in this behavior or not.
